### ISSUE TITLE
掲載する非公式クライアントの範囲

### ISSUE THREAD

**Issue by myConsciousness** [bluesky-jp/welcome-bluesky#5]
単純に公式atprotoの[Community Projects](https://atproto.com/community/projects#clients)に掲載されてるサードパーティクライアントを掲載してもいいのですが、せっかく日本人向けのWebサイトなので日本人が作ったアプリを優先して掲載してもいいのかも。

- https://atproto.com/community/projects#clients

**Comment by uakihir0** [comment]
そうですね、あと特にこのドキュメントについてなんですが、 **公式アプリが英語で分かりにくい** という人に向けている側面もあるので、特に日本語化されているか、をしっかり書ければいいかなと思ってます。
